src/design/optForceGAMS/optForceWithGAMS.m

```diff
@@ -0,0 +1,840 @@
+%
+% targetRxn (obligatory):   Type: string
+%                           Description: string containing the ID for the
+%                           reaction whose flux is intented to be increased.
+%                           For example, if the production of succionate is
+%                           desired to be increased, 'EX_suc' should be
+%                           chosen as the target reaction
+%                           Example: targetRxn='EX_suc';
+%
+% mustU (obligatory):       Type: cell array.
+%                           Description: List of reactions in the MustU set
+%                           This input can be obtained by running the
+%                           script findMustU.m
+%                           Alternatively, there is a second usage of this
+%                           input:
+%                           Type: string.
+%                           Description: name of the .xls file containing
+%                           the list of the reactions in the MustU set
+%                           Example first usage: mustU={'R21_f';'R22_f'};
+%                           Example second usage: mustU='MustU';
+%
+% mustL (obligatory):       Type: cell array.
+%                           Description: List of reactions in the MustL set
+%                           This input can be obtained by running the
+%                           script findMustL.m
+%                           Alternatively, there is a second usage of this
+%                           input:
+%                           Type: string.
+%                           Description: name of the .xls file containing
+%                           the list of the reactions in the MustU set
+%                           Example first usage: mustL={'R11_f';'R26_f'};
+%                           Example second usage: mustL='MustL';
+%
+% minFluxesW (obligatory):   Type: double array of size n_rxns x1
+%                            Description: Minimum fluxes for each reaction
+%                            in the model for wild-type strain
+%                            Example: minFluxesW=[-90; -56];
+%
+% maxFluxesW (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Maximum fluxes for each reaction
+%                            in the model for wild-type strain
+%                            Example: maxFluxesW=[92; -86];
+%
+% minFluxesM (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Minimum fluxes for each reaction
+%                            in the model for mutant strain
+%                            Example: minFluxesW=[-90; -56];
+%
+% maxFluxesM (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Maxmum fluxes for each reaction
+%                            in the model for mutant strain
+%                            Example: maxFluxesW=[92; -86];
+%
+% k(optional):              Type: double
```

Comment:
I agree. In my new pull request (https://codecov.io/gh/opencobra/cobratoolbox/pull/523?src=pr&el=desc) I removed all the files generated during the execution of functions. 

About the description of the output files, In the new PR you will see that I put the description of outputs files inside of each function, in the OUTPUT section. Additionally, In the tutorials, I added comments to highlight the lines in which the files are generated. Is that what you meant, isn't it?